release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -102,6 +102,21 @@ With {product-title} {product-version}, running `oc describe` on an image now re
 [id="ocp-4-13-images"]
 === Images
 
+[id="ocp-4-13-auth"]
+=== Authentication and authorization
+
+[id="ocp-4-13-psa-tp-feature-set"]
+==== Pod security admission restricted enforcement (Technology Preview)
+
+With this release, pod security admission restricted _enforcement_ is available as a Technology Preview feature by enabling the `TechPreviewNoUpgrade` feature set. If you enable the `TechPreviewNoUpgrade` feature set, pods are rejected if they violate pod security standards, instead of only logging a warning.
```

Comment:
If you know what part of the platform rejects the pods, consider making the second sentence active and moving the two potential consequences closer together, perhaps like:
```suggestion
With this release, pod security admission restricted _enforcement_ is available as a Technology Preview feature by enabling the `TechPreviewNoUpgrade` feature set. If you enable the `TechPreviewNoUpgrade` feature set, if pods violate pods security standards, the <component> rejects them instead of only logging a warning.
```

Reply:
I'll leave it as is, since I don't think the component details are important enough to call out.